client/txn_test.go

```diff
@@ -101,12 +109,11 @@ func newTestSender(pre, post func(proto.Call)) SenderFunc {
 func TestTxnRequestTxnTimestamp(t *testing.T) {
 	defer leaktest.AfterTest(t)
 	makeTS := func(walltime int64, logical int32) proto.Timestamp {
-		return proto.Timestamp{
-			WallTime: walltime,
-			Logical:  logical,
-		}
+		return proto.ZeroTimestamp.Add(walltime, logical)
```

Comment:
but there is: ![](https://avatars3.githubusercontent.com/u/1535036?v=3&s=20)

:dragon_face: